packages/components/src/Tile/InformationTile/_docs/InformationTile.stories.tsx

```diff
@@ -9,6 +9,7 @@ const meta = {
     title: "Title",
     metadata: "Side A",
     footer: <>Example Footer</>,
+    information: "Side B",
```

Comment:
I reckon we move this into the args in ln 35 just to make it explicit that it relates to this story

Reply:
We have it in both places atm, is it needed by all the tests then?

Reply:
Done!